content/guides/understanding-los-records.md

```diff
@@ -0,0 +1,129 @@
+# Understanding LOS Records
+
+## What is LOS?
+
+LOS stands for Length Of Stay and LOS Records are a simple but powerful way to define rates.
+
+It's best explained through examples.
```

Comment:
I would actually explain it just to have some idea what to expect from the examples. Something like: 
```
LOS Records are precomputed cache of prices for consecutive days for a given day. 
```